app/api/v2/models/Basemodel.py

```diff
@@ -0,0 +1,20 @@
+# from app.api.v2.utils.database import MyDatabase
+
+# execCommand = MyDatabase.ConnectDatabase
+
+
+# class DBManipulateBase():
+#     def CreateTable(self, query):
+#         execCommand(query)
+
+#     def CreateRow(self, query):
+#         execCommand(query)
+
+#     def Update_Row(self, query):
+#         execCommand(query)
+
+#     def Read_Row_all(self, query):
+#         execCommand(query)
+
+#     def Read_Row_one(self, query):
+#         execCommand(query)
```

Comment:
If you are not using this file you should delete it. A commented code should not be pushed